docs/source/upgrade-guides/workarea-3-5-0.html.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: Upgrading from Workarea v3.4 to v3.5
-excerpt: A catalog of changes to watch out for while upgrading between minor versions
+title: Workarea 3.5.0 Upgrade Guide
+excerpt: Upgrade guide for Workarea 3.5.0
```

Comment:
Is this a placeholder or will this `excerpt` give a bit more info than the title?